src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,251 @@
+    const { subscribedEmail, error } = await onSubscribe(emailAddress)
+
+    if (error) {
+      onError()
+    }
+    onSubscribeSuccess(subscribedEmail)
+  }, [emailAddress, onError, onSubscribe, onSubscribeSuccess])
+
+  const handleReSendEmail = useCallback(async () => {
+    const { error } = await onReSendVerificationEmail()
+
+    if (error) {
+      onError()
+    }
+  }, [onError, onReSendVerificationEmail])
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        padding-top: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <img src={emailIllustration} />
+        <span
+          css={`
+            ${textStyle('title2')};
+            margin-top: ${4 * GU}px;
+          `}
+        >
+          Verify your email address
+        </span>
+
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-top: ${1.5 * GU}px;
+          `}
+        >
+          {`Almost there! We’ve sent a verification email to ${email}.
+          Kindly check your inbox and click the link to verify your account. ${
+            updateMode
+              ? 'Alternatively, you can update this email or delete it, if you wish to unsubscribe'
+              : ''
+          }`}
+        </span>
+        {(() => {
+          if (updateMode) {
+            const buttonWidth = compactMode
+              ? '100% '
+              : `calc((100% - ${2 * GU}px) /  2)`
+            return (
+              <>
+                <div
+                  css={`
+                    margin-top: ${5 * GU}px;
+                  `}
+                >
+                  <Field label="Update email address">
+                    <TextInput
+                      value={emailAddress}
+                      adornment={
+                        emailInvalid === false ? (
+                          <IconCheck
+                            css={`
+                              color: ${theme.positive};
+                            `}
+                          />
+                        ) : emailAddress.trim() ? (
+                          <IconCross
+                            css={`
+                              color: ${theme.negative};
+                            `}
+                          />
+                        ) : (
+                          <IconCheck
+                            css={`
+                              opacity: 0;
+                            `}
+                          />
+                        )
+                      }
+                      adornmentPosition="end"
+                      type="email"
+                      wide
+                      onChange={handleEmailAddressChange}
+                      placeholder="email@address.com"
```

Comment:
Nitpicking: I generally use [example.org](http://example.org/) for examples as it is reserved for this case. Also should we use a name? Like `rodrigo` :laughing: 

```suggestion
                      placeholder="rodrigo@example.org"
```

Reply:
ahh haha just using the same that is on the designs.
https://www.figma.com/file/IdVA6iMn0cWLV5imylwEUP/%E2%9A%96%EF%B8%8F-Court-Dashboard-Master?node-id=4980%3A7830